lib/simulation/openfisca/mapping/ressources.js

```diff
@@ -17,6 +17,11 @@ exports.individu = {
         src: 'revenusSalarie',
         fn: function(value) { return value / 0.78; }
     },
+    sali: {
+        src: 'revenusSalarie',
+        // net imposable = net + CRG + CRDS = net + (tauxCRG/CRDS) * brut
+        fn: function(value) { return value + 0.029 * 0.9825 * value / 0.78; }
```

Comment:
If the `value` is the `src`, call the parameter `revenusSalaries` rather than `value`, that will make the computation more explicit  :)